### PR TITLE
Add a config property to ignore paths.

### PR DESCRIPTION
**Pull request by dashea**
ignoreList is a list of names to filter out of the directory and file
lists when automatically determining which files to lint.

### CODE REVIEW

File: pocketlint/__init__.py [hunk 1]
```diff
@@ -94,6 +94,15 @@ def pylintPlugins(self):
...
                ]
 
+    @property
+    def ignoreList(self):
+        """A list of names to skip when automatically determining the list of
+           files to lint. The items in this list could be a particular filename
+           to skip or a directory that the linter should not traverse into.
+           The items should be just basenames, not paths.
+        """
+        return []
```
Comment (vpodzime) [inline pocketlint/__init__.py]: What about making this a set?

**Comment by vpodzime** [comment]
Looks good to me other than the suggestion above.

File: pocketlint/__init__.py (hunk 1 above)
Comment (dashea) [inline pocketlint/__init__.py]: Sure, that would work

**Comment by clumens** [comment]
We already have "# pylint: skip-file" that you can add directly into a file, so this does overlap a little bit.  However, I like the directory piece too so I'll take it.

**Pull request merged**